packages/aila/src/utils/lessonPlan/fetchLessonPlanContentById.ts

```diff
@@ -23,6 +23,11 @@ export async function fetchLessonPlanContentById(
   const parsedPlan = tryWithErrorReporting(
     () => LessonPlanSchemaWhilstStreaming.parse(lessonPlanRecord.content),
     "Failed to parse lesson plan content",
+    undefined,
+    undefined,
```

Comment:
This is one of the reasons I nearly always use objects as a single argument to functions. Extending the signature beyond option args is horrible otherwise.